packages/postgresql/data_stream/statement/manifest.yml

```
@@ -18,7 +18,6 @@ streams:
         multi: false
         required: false
         show_user: false
-        description: >
-          Processors are used to reduce the number of fields in the exported event or to enhance the event with metadata. This executes in the agent before the events are shipped. See [Processors](https://www.elastic.co/guide/en/fleet/current/elastic-agent-processor-configuration.html) for details.  
+        description: "Processors are used to reduce the number of fields in the exported event or to enhance the event with metadata. This executes in the agent before the events are shipped. See [Processors](https://www.elastic.co/guide/en/fleet/current/elastic-agent-processor-configuration.html) for details.  \n"
```

```suggestion
        description: "Processors are used to reduce the number of fields in the exported event or to enhance the event with metadata. The agent executes this before the events are shipped. See [Processors](https://www.elastic.co/guide/en/fleet/current/elastic-agent-processor-configuration.html) for details.\n"
```

Not sure why did we add "\n" here? Any particular reason?